apps/sps-ordbokene-web/components/Inflection/mixins/ordbankUtils.js

```diff
@@ -52,10 +56,11 @@ function appendTwoWordForms (wf1, wf2) {
         let res = wf1.map(w => w)
         wf2.forEach(w => { if (!wf1.find(x => x == w)) { res.push(w) } })
     }
+    if (Array.isArray(res)) res = res.sort((a,b) => a.localeCompare(b))
```

Comment:
I had problems with localecompare elsewhere in the codebase because it used a different locale on the server than in the client. It caused a rendering error where content was duplicated. It might not be an issue any more though

Reply:
Will it create any conflict with your version if I merge this?

Reply:
No, but there is a chance it may cause a rendering issue. I din't find any rendering issues in the preview, so the bug might have been fixed in Nuxt